public/index.php

```diff
@@ -1,20 +1,37 @@
 <?php
-
+use App\Router;
+use App\Controllers\AuthController;
+use App\Controllers\NewsController;
 use App\Session;
-use App\Config\Config;
+use App\EnvLoader;
+use App\Controllers\ErrorController;
 
 require '../vendor/autoload.php';
 
-if ((new Config)->get('DEBUG_MODE') == 'true') {
+if ((new EnvLoader(__DIR__ . '/../.env'))->get('DEBUG_MODE') == 'true') {
```

Comment:
You got a helper for this ;)

```suggestion
if (env('DEBUG_MODE') === 'true') {
```